lib/mongorito.js

```diff
@@ -866,9 +888,9 @@ exports = module.exports = Mongorito;
 
 exports.Model = Model;
 
-Object.keys(mongoskin).forEach(function (key) {
+Object.keys(mongodb).forEach(function (key) {
   if (['connect', 'version', 'db'].indexOf(key) === -1) {
-    exports[key] = mongoskin[key];
+    exports[key] = mongodb[key];
```

Comment:
It should not be just driver replacement, we need to ensure we expose useful properties from mongodb. `['connect', 'version', 'db']` was ignored when using mongoskin. What properties does `mongodb` expose?